interfaces/IDRM.h

```diff
@@ -459,9 +459,7 @@ class SystemFactoryType : public ISystemFactory {
         , _instance()
     {
     }
-    virtual ~SystemFactoryType()
```

Comment:
We nowadays make virtual destructors that are in a derived class override. 
So please make the above ~SystemFactoryType() override = default;
Please check for other instances where that is applicable as well in the PR

Reply:
Updated